builtins/amp-ad.js

```diff
@@ -268,6 +268,9 @@ export function installAd(win) {
             this.iframe_.style.visibility = '';
             this.sendEmbedInfo_(this.isInViewport());
           }, /* opt_is3P */ true);
+          viewerFor(this.getWin()).onVisibilityChanged(() => {
```

Comment:
Not generally necessary, but since there were now 3 of these in the file I cleaned it up.